### DISCUSSION TITLE
onPaid callback not working in test mode?

### DISCUSSION THREAD

**Discussion by ggcaponetto** [Glench/ExtPay#124] [Q&A] [answered]
Is the followind callback expected to work in test mode?

```js
extpay.onPaid.addListener((user) => {
        console.log("user paid.. reloading the page.", user);
        window.location.reload();
      });
```

**Comment by Glench** [comment]
It works fine in test mode (see the sample extension). If you provide your manifest.json and other relevant details then maybe we can track it down

**Comment by ggcaponetto** [comment, marked as answer]
Got it working. Thanks for highlighting the sample.

**Reply by Glench** [reply]
Excellent!
